cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -245,7 +245,7 @@ func ValidateIPNetFromString(subnet string, minAddrs int64, fldPath *field.Path)
 }
 
 // ValidateNetworking validates networking configuration
-func ValidateNetworking(c *kubeadm.Networking, fldPath *field.Path) field.ErrorList {
+func ValidateNetworking(c *kubeadm.Networking, _ *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	allErrs = append(allErrs, apivalidation.ValidateDNS1123Subdomain(c.DNSDomain, field.NewPath("dns-domain"))...)
```

Comment:
Typical use would be that the other fields in this (such as 'dns-domain', 'pod-subnet') would be `fldPath.Child` of the one passed in. Not sure the reason for it not to be, or if the best-practices have shifted.